src/airstack/onchain_graph.py

```diff
@@ -0,0 +1,759 @@
+"""
+Module: onchain_graph.py
+Description: This module contains the methods of Onchain Graphs.
+"""
+
+from airstack.execute_query import AirstackClient
+from airstack.generic import format_poaps_data, format_farcaster_followings_data, format_lens_followings_data, format_farcaster_followers_data, format_lens_followers_data, format_token_sent_data, format_token_received_data, format_eth_nft_data, format_polygon_nft_data, calculating_score
+
+
+class ExecuteOnchainGraph():
+    """Class to store onchain graph function
+    """
+
+    def __init__(self, url=None, api_key=None):
+        self.url = url
+        self.api_client = AirstackClient(api_key=api_key)
+
+    async def __fetch_poaps_data(self, address, existing_users=[]):
+        user_poaps_event_ids_query = """
```

Comment:
These queries to be moved to the configuration file. Let's keep the code file compact